CRVResponse/src/CrvCoincidenceCheck_module.cc

```diff
@@ -459,10 +460,10 @@ if(crvCoincidenceCollection->size()==0)
     size_t n=layerIterators.size();
     if(n==0) return;
 
-    double maxTimeDifferences[n];
-    double times[n];
-    double timesPulseStart[n], timesPulseEnd[n];
-    double x[n], y[n];
+    std::vector<double> maxTimeDifferences(n);
+    std::vector<double> times(n);
+    std::vector<double> timesPulseStart(n), timesPulseEnd(n);
+    std::vector<double> x(n), y(n);
```

Comment:
This module is not used anymore. I kept it just in case some older scripts still use it.

Reply:
Thanks Ralf.